lib/transactional.pm

```diff
@@ -72,7 +79,7 @@ sub check_reboot_changes {
     die "Change expected: $change_expected, happeed: $change_happened" if $change_expected != $change_happened;
```

Comment:
not your code but could you fix typo s/happeed/happened/

Reply:
fixed